package.json

```diff
@@ -17,11 +17,12 @@
     "posttest": "npm run eslint",
     "prepublish": "npm run lib",
     "jest": "node_modules/.bin/jest",
+    "jest-watch": "node_modules/.bin/jest --watch",
     "eslint": "node_modules/.bin/eslint src/**/*.js",
     "test": "npm run unit-test && npm run jest",
     "clean": "rm -rf lib && mkdir lib",
     "lib": "npm run clean && babel src -d lib",
-    "docs": "node ./scripts/docs-server",
+    "docs-server": "node ./scripts/docs-server",
```

Comment:
Ah, sorry, should have been more clear here. Please leave both `docs` and `docs-server` but have the docs script just call the docs-server one: 
```
"docs": "npm run docs-server",
```